### DISCUSSION TITLE
Add support for UV (Python virtual env)

### DISCUSSION THREAD

**Discussion by voltuer** [JanDeDobbeleer/oh-my-posh#5629] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What would you like to see added?

Hello. I use [UV](https://astral.sh/blog/uv) to manage my Python projects, which is very popular (31k stars in github)

When I'm on WSL, ohmyzsh recognizes it perfectly:
<img width="436" alt="image" src="https://github.com/user-attachments/assets/993682ce-9d5b-4b78-a7b3-b36a9e7760ee">

But when I'm on PWSH it does not:
<img width="365" alt="image" src="https://github.com/user-attachments/assets/9ae68d24-512f-44a5-9c14-6e00583e5f1d">

The main problem with this, beside the bad UI experience, is that when the prompt is long, it moves down and takes 2 lines:
<img width="626" alt="image" src="https://github.com/user-attachments/assets/6677c3ea-9969-4eb6-8f97-321baf399688">

I'd love to fix this myself but I can't figure out how. I think the ideal panorama would be to show the name besides the python icon and not at the begining of the line, like OMZ does, and make it use just 1 line

**Comment by lewis-yeung** [comment]
Does setting `export VIRTUAL_ENV_DISABLE_PROMPT=1` resolve the issue?

**Comment by voltuer** [comment]
> Does setting `export VIRTUAL_ENV_DISABLE_PROMPT=1` resolve the issue?

@lewis-yeung i don't know how to apply that, i'm using powershell and ´export´ is not a valid command

**Comment by lewis-yeung** [comment]
> when I'm on PWSH

@voltuer Aha, I didn't notice that. The command is just setting an environment variable. For PowerShell you should add `$env:VIRTUAL_ENV_DISABLE_PROMPT = 1` to your profile.

**Comment by JanDeDobbeleer** [comment]
@voltuer as @lewis-yeung stated, you need to disable the default built-in prompt. Next up is adding the [Python](https://ohmyposh.dev/docs/segments/languages/python) segment as the default config doesn't have that one. Have a look at the [documentation](https://ohmyposh.dev/docs/installation/customize#custom-configuration) on how to configure that.
